### DISCUSSION TITLE
please help me , my chia was lost

### DISCUSSION THREAD

**Discussion by Hosseinkrn** [Chia-Network/chia-blockchain#8839] [Q&A]
hi chief , i sent 0.4958 chia coin from my exchange ((xch18qmny7yek6gvy6j6zdkzfqcgegkta4s6r6q8cx2ky0guk39lkvzspstnj8) to my chia wallet application on my computer ( xch1kwdwtt2w737vud4lzvxr7wag5ky8dyv5f5fa65wn5uwff054r4ksd2nnl4)  but i dident have any chia in my wallet , i send you some picture , please help me , i need that :)

![Screenshot (145)](https://user-images.githubusercontent.com/92608238/137556687-1fdeb6e6-a1fc-4b23-b3a3-66acffaa7675.png)
![Screenshot (146)](https://user-images.githubusercontent.com/92608238/137556698-8d6410f0-e6a5-45ae-8dd3-acf6efed636a.png)
![Screenshot (147)](https://user-images.githubusercontent.com/92608238/137556710-aa39a709-46fa-493c-8fed-b113d77fe713.png)

**Comment by marssystems** [comment]
Your wallet isn't synced. Wait for your wallet to sync and you will get your XCH.

**Reply by Hosseinkrn** [reply]
how i can sync it ?

**Comment by BrandtH22** [comment]
@Hosseinkrn , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
